apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/spi/defaultimpl/DefaultEmailService.java

```diff
@@ -64,21 +68,32 @@ public void send(Email email) {
       String user = portalConfig.emailConfigUser();
       String password = portalConfig.emailConfigPassword();
 
-      t = (SMTPTransport) session.getTransport("smtp");
-      t.connect(host, user, password);
-      msg.saveChanges();
-      t.sendMessage(msg, msg.getAllRecipients());
-      logger.debug("email response: {}", t.getLastServerResponse());
+      if (useSsl) {
```

Comment:
It looks like unnecessary to declare a new variable `tSsl`. Could we consider streamlining the code even further, as suggested below?

```java
String protocol = useSSl ? "smtps" : "smtp";
t = (SMTPTransport) session.getTransport(protocol);
```